python/lsst/meas/astrom/matchOptimisticB.py

```diff
@@ -259,46 +275,62 @@ def matchObjectsToSources(self, refCat, sourceCat, wcs, refFluxField):
                             int(self.config.minFracMatchedPairs * min([len(refCat), len(goodSources)])))
 
         # match clean (possibly saturated) sources and then purge saturated sources from the match list
+#        matches0 = self._doMatch(
```

Comment:
I completely removed the matching with potentially saturated sources. I think that given the weaknesses of the fitter, a saturated source has a big chance to screw things up  due to a badly measured position